### ISSUE TITLE
bug(linux): IBus message appears in the terminal window while installing a Keyboard using Configuration Window

### ISSUE THREAD

**Issue by bharanidharanj** [keymanapp/keyman#8453]
### Describe the bug

I noticed that while installing a keyboard using Configuration dialog, IBUS message appears line by line in the terminal window which I have already opened. 

### Reproduce the bug

1. In Ubuntu 22.04 (Jammy Jelly fish Linux OS) install the latest Keyman 17.0.68 alpha test
2. Open Terminal window.
3. Run km-config
4. We may notice that the Keyman Configuration dialog opens. 
5. Click Download button. 
6. Here, I noticed that an 'IBUS-Message' appear on the terminal window. 
7. Install any keyboard. (eg., Khmer Angkor) . 
8. Here, I noticed that the sequence of lines appears (related to IBUS-Message)  on the terminal window. 

Note: I am not seeing this message in Ubuntu 22.10 Kudu Linux OS. 

..IBUS - Message appears in Ubuntu 22.04 Linux OS

   <img width="300" src="https://user-images.githubusercontent.com/19683143/225637403-99f4d62f-d64e-4060-850d-d0cfa4b4eb94.PNG">

..IBUS - Message did not appear in Ubuntu 22.10 Linux OS

   <img width="300" src="https://user-images.githubusercontent.com/19683143/225637459-90b4e9f6-65d5-4896-b80d-63f146382604.PNG">

### Expected behavior

It would be better to not appear in the terminal window in Ubuntu 22.04 Linux OS. 

### Keyman apps

- [ ] Keyman for Android
- [ ] Keyman for iPhone and iPad
- [X] Keyman for Linux
- [ ] Keyman for macOS
- [ ] Keyman for Windows
- [ ] Keyman Developer
- [ ] KeymanWeb
- [ ] Other - give details at bottom of form

### Keyman version

17.0.68-alpha

### Operating system

Ubuntu 22.04 Jammy Jelly Fish Linux OS

**Comment by ermshiperete** [comment]
This should be fixed in ibus 1.5.28-2sil1. I uploaded that change to keyman-test ppa; after it's tested I can make it more widely available.

# User Testing

## Preparations

- The tests should be run on these Linux platforms:
  - **GROUP_FOCAL**: Ubuntu 20.04 Focal with Gnome Shell and X11
  - **GROUP_JAMMY_X11**: Ubuntu 22.04 Jammy with Gnome Shell and X11
  - **GROUP_KINETIC_X11**: Ubuntu 22.10 Kinetic with Gnome Shell and X11
  - **GROUP_WASTA**: Wasta 20.04 with Cinnamon

- Add the keyman-test ppa:

  ```bash
  sudo add-apt-repository ppa:keymanapp/keyman-test
  ```

- Install updates:

  ```bash
  sudo apt update
  sudo apt upgrade
  ```

- verify that this installed the patched ibus version:

  ```bash
  dpkg -l ibus | grep ibus
  ```

  should show a version number starting with **1.5.28-2sil1**

- Install Keyman alpha version

- Reboot

## Tests

- **TEST_KB_INSTALL**: open `km-config` from the command line, install a Khmer Angkor keyboard. Verify that no output is shown in terminal window.
- **TEST_KM_WRITER:** Open LibreOffice Writer. Switch to "Khmer Angkor" keyboard. Type `xEjmr`. Verify that the output is "ខ្មែរ".
- **TEST_KM_GEDIT:** Open gedit. Switch to "Khmer Angkor" keyboard. Type `xEjmr`. Verify that the output is "ខ្មែរ".
- **TEST_CONTEXT:**
  - Open gedit
  - Switch to "IPA (SIL)" keyboard.
  - Type
    ```
    an
    m
    ```
  - click after `n`
  - type `>`
  - verify that the output is
    ```
    aŋ
    m
    ```

**Comment by mcdurdin** [comment]
> ibus 1.5.28-3sil1

Is this a fix we've made to ibus that should be contributed upstream, or an updated version of ibus?

**Comment by bharanidharanj** [comment]
### GROUP_FOCAL: Ubuntu 20.04 Focal with Gnome Shell and X11

* **TEST_KB_INSTALL (PASSED):** Tested this as per the instructions and in Ubuntu 20.04 Focal linux OS (VM) and here is my observation: 1. Able to see the ibus version number 1.5.28-3sil1~focal amd 64 in the terminal. 2. Installed the latest Keyman 17.0.83-alph a build. 3. Installed Khmer Angkor Keyboard. Verified that no output is shows in the terminal windoww.

   <img width="300" src="https://user-images.githubusercontent.com/19683143/230875205-6482ebd0-dc5b-4286-b53a-bc933ea698a1.PNG">

* **TEST_KM_WRITER (PASSED):** Verified that the output shows ខ្មែរ in the LibreOffice Writer.
   
   <img width="300" src="https://user-images.githubusercontent.com/19683143/230885854-e0a31b0f-0ace-4c87-831d-7c46843e04ad.PNG">


* **TEST_KM_GEDIT (PASSED):** Verified that the output shows ខ្មែរ in the Text Editor. 

   <img width="300" src="https://user-images.githubusercontent.com/19683143/230885973-99f9f514-2ad0-4f7d-b6e7-6193491f6ce4.PNG">

* **TEST_CONTEXT (PASSED):** 1. Installed IPA (SIL) keyboard. 2. Verified that the output shows aŋ after typing > in the text editor. 

   <img width="300" src="https://user-images.githubusercontent.com/19683143/230886037-37a8e5c7-2c61-4d64-a29a-524ebc77678f.PNG">

**Comment by bharanidharanj** [comment]
### GROUP_JAMMY_X11: Ubuntu 22.04 Jammy with Gnome Shell and X11

* **TEST_KB_INSTALL (PASSED):** Tested this as per the instructions and in Ubuntu 20.04 Focal linux OS (VM) and here is my observation: 1. Able to see the ibus version number 1.5.28-3sil1~jammy amd64 in the terminal. 2. Installed the latest Keyman 17.0.83-alph a build. 3. Installed Khmer Angkor Keyboard. Verified that no output is shows in the terminal window.
   
   <img width="200" src="https://user-images.githubusercontent.com/19683143/230891089-6adf57c2-202f-4eba-a8ac-0f8c8eefaebb.PNG">


* **TEST_KM_WRITER (FAILED):** Noticed that typing xEjmr did not show the expected output in the LibreOffice Writer. Seems to be an issue.

   <img width="200" src="https://user-images.githubusercontent.com/19683143/230891113-f08f61b8-6332-4f51-886a-6fd10f403c73.PNG">


* **TEST_KM_GEDIT (PASSED):** Verified that the output shows ខ្មែរ in the Text Editor. 

   <img width="200" src="https://user-images.githubusercontent.com/19683143/230891127-9b7e1ba3-0f5f-499d-ad4a-3dcf954442e3.PNG">


* **TEST_CONTEXT (PASSED):** 1. Installed IPA (SIL) keyboard. Verified that the output shows aŋ after typing > in the text editor.
   
   <img width="200" src="https://user-images.githubusercontent.com/19683143/230891142-97948ee3-5185-4150-8827-c3a58e2ea2aa.PNG">

**Comment by bharanidharanj** [comment]
### GROUP_KINETIC_X11: Ubuntu 22.10 Kinetic with Gnome Shell and X11

* **TEST_KB_INSTALL (PASSED):** Tested this as per the instructions and in Ubuntu 22.10 Kinetic Kudu linux OS (VM) and here is my observation: 1. Able to see the ibus version number 1.5.28-3sil1~kinetic amd 64 in the terminal. 2. Installed the latest Keyman 17.0.83-alph a build. 3. Installed Khmer Angkor Keyboard. Verified that no output is shows in the terminal window.

   <img width="300" src="https://user-images.githubusercontent.com/19683143/230895570-38b9ccc9-d30b-4692-a397-42bd3ff2f764.PNG">


* **TEST_KM_WRITER (PASSED):** Verified that the output shows ខ្មែរ in the LibreOffice Writer. 

   <img width="200" src="https://user-images.githubusercontent.com/19683143/230895605-f92e6165-6f6d-4449-962f-0dd07e0a2e5b.PNG">


* **TEST_KM_GEDIT (PASSED):** Verified that the output shows ខ្មែរ in the Text Editor.

   <img width="200" src="https://user-images.githubusercontent.com/19683143/230895650-d557d7b2-5421-47e1-8ae4-a51b6d41b3fc.PNG">


* **TEST_CONTEXT (PASSED):** 1. Installed IPA (SIL) keyboard. 2. Verified that the output shows aŋ after typing > in the text editor. 

   <img width="200" src="https://user-images.githubusercontent.com/19683143/230895689-5ceaae6b-cdc2-4d9a-b76c-8b8c3bf14f72.PN">

**Comment by ermshiperete** [comment]
> > ibus 1.5.28-3sil1
> 
> Is this a fix we've made to ibus that should be contributed upstream, or an updated version of ibus?

That was some left-over debug code in our patched version of ibus. I removed that and also updated to the current upstream version of ibus.

**Comment by ermshiperete** [comment]
All tests pass if the Khmer Monudlkiri gets used: in Writer, select all text and apply "Khmer Mondulkiri" font. In Gedit in Preferences, tab "Font & Colors" select "Editor font": "Khmer Mondulkiri Regular".

###  GROUP_JAMMY_X11
- TEST_KM_WRITER(PASSED)

### GROUP_WASTA
- TEST_KM_WRITER(PASSED)
- TEST_KM_GEDIT(PASSED)

**Comment by ermshiperete** [comment]
###  GROUP_JAMMY_X11
- TEST_KM_WRITER(PASSED)

**Comment by ermshiperete** [comment]
## GROUP_JAMMY_X11
- TEST_KM_WRITER(PASSED)

**Comment by ermshiperete** [comment]
Since all tests pass with the correct fonts I'm closing this issue. Fixed with ibus 1.5.28-3sil1.
